GetJobber/atlantis
# chore: Audit Atlantis packages

## Motivations

Atlantis packages are getting pretty far out of date. This PR will do an audit of vulnerabilities for `most` of the Atlantis npm packages.

There are some packages that need to be audited outside of this PR as they have a much much larger impact.

**Notes**

Updating the `typed-css-modules` caused a change to the way that css types are made. This causes changes in every `.css.d.ts`.

As part of this I have also ignored the `.css.d.ts` files in the `eslintignore` file as these are auto generated and we shouldn't care about linting them.

## Changes



### Added

- 

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

- pull down branch
- run all the `npm` scripts
- see they all work

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

![Random photo of Atlantis](https://loremflickr.com/672/400/atlantis)
